### DISCUSSION TITLE
Problem following when using multiple accounts

### DISCUSSION THREAD

**Discussion by glyn** [tuskyapp/Tusky#2276] [Q&A] [answered]
I have two accounts on distinct servers. I can log into them both on Tusky just fine and see my posts, notifications, etc. But I cannot follow users from the second account I added. I can however follow the same users using the web interface, so I'm wondering if there is an issue with Tusky.

When I go to a particular user's profile from the second account I added to Tusky, no "Follow" button is displayed, although I briefly see a pop-up saying "An error occurred" with the option to retry. The pop-up disappears after a couple of seconds and there is no sign of an error code.

Anyone else seeing this?

**Comment by connyduck** [comment, marked as answer]
That sounds a lot like [this bug](https://git.qoto.org/qoto/qoto/-/issues/11)

On which instance is your second account?

**Reply by glyn** [reply]
> That sounds a lot like [this bug](https://git.qoto.org/qoto/qoto/-/issues/11)


Yes, that seems plausible.
> 
> On which instance is your second account?

qoto.org

**Comment by connyduck** [comment]
This will be fixed in Tusky 16 - https://github.com/tuskyapp/Tusky/pull/2298

**Reply by glyn** [reply]
Great stuff. Thanks!
